Modules/Relationships/test/Relationships.Domain.Tests/TestHelpers/TestData.cs

```diff
@@ -70,4 +70,13 @@ public static Relationship CreateDecomposedRelationship(IdentityAddress? from =
         relationship.Decompose(decomposedBy, DEVICE_1);
         return relationship;
     }
+
+    public static Relationship CreateRelationshipInDecompositionByFirstParticipant(IdentityAddress from, IdentityAddress to)
```

Comment:
1. It should be "DecomposedBy" instead of Decomposition, right?
2. From the method signature you cannot see which participant is meant with "FirstParticipant". So I'd suggest to call it "...ByFrom"

Summary:

```suggestion
    public static Relationship CreateRelationshipInDecomposedByFrom(IdentityAddress from, IdentityAddress to)
```

Reply:
I agree completely